packages/lesswrong/server/languageModels/modGPT.ts

```diff
@@ -109,42 +109,17 @@ async function checkModGPT(comment: DbComment): Promise<void> {
     nonTextTags: ['img', 'style']
   })
   const text = htmlToText(html)
-
-  let response = await getModGPTAnalysis(api, text)
-  // If the API is too busy the first time, we try one more time.
-  // See https://platform.openai.com/docs/guides/error-codes/api-errors
```

Comment:
Here I got rid of the retry since it was never used anyway. Maybe it was just because it was checking for the wrong error code? Anyway this should be functionally the same.